src/main/java/org/homeschoolpebt/app/inputs/Pebt.java

```diff
@@ -85,7 +85,10 @@ public class Pebt extends FlowInputs {
   @NotBlank
   private String householdMemberLastName;
   private String householdMemberReceivesBenefits;
-  private String householdMemberBenefitsCaseNumber;
+  @Size(min=1, max=15, message="{household-receives-benefits.calfresh-provide-1-15-digits}")
+  private String householdMemberBenefitsCaseNumberCalfresh;
+  @Size(min=1, max=15, message="{household-receives-benefits.calworks-provide-1-15-digits}")
+  private String householdMemberBenefitsCaseNumberCalworks;
   private String householdMemberBenefitsCaseNumberFDPIR;
```

Comment:
Huh, why do we need a custom validator for FDPIR? Is it because we would normally use a `@NotBlank` here but we can't because it's conditional on `householdMemberReceivesBenefits` equaling `FDPIR`?